pkg/operation/botanist/kubeapiserverexposure_test.go

```diff
@@ -139,42 +139,5 @@ var _ = Describe("KubeAPIServerExposure", func() {
 				Entry("NodePort", corev1.ServiceTypeNodePort),
 			)
 		})
-
-		Context("sni disabled", func() {
```

Comment:
As mentioned earlier, let's keep this for another while.

Reply:
Unlocked APIServerSNI and added the tests again.

Reply:
I think you should still lock it since the feature gate only exists in `gardenlet` and not in `gardener-operator`.

Reply:
But the tests won't work if it's locked.

Reply:
Ah, got it. Sorry, I was confused then. This is about the `botanist` and not the `kubeapiserverexposure` package. Then it should be fine to drop the tests here.